components/event/EventKey.js

```diff
@@ -1,7 +1,12 @@
 import { IconContext } from "react-icons";
 import { MdOutlineOnlinePrediction, MdOutlinePeople } from "react-icons/md";
 
-export default function EventKey() {
+export default function EventKey({ events }) {
+  const inPersonEvents = events.filter(event => event.isInPerson)
+    .map(inPersonEvent => inPersonEvent)
```

Comment:
Is the map needed, could it just be `events.filter(event => event.isInPerson).length`?

Reply:
Not at all the map was pretty useless indeed. I was getting back undefined at first and I thought the map was fixing this.